react/testing_4/src/redux/Food/actionsTestWithRealStore.test.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { createStore, applyMiddleware, compose } from "redux";
+import thunk from "redux-thunk";
+import { Provider } from "react-redux";
+import { configure, mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import FoodList from "../../app/screens/Home/components/FoodList";
+
+import actionCreators from "./actions";
+import reducer from "./reducer";
+
+configure({ adapter: new Adapter() });
+
+describe("Test - Store + reducer integration testing with real store", async () => {
+  let store, wrapper;
+
+  beforeEach(() => {
+    store = createStore(reducer, compose(applyMiddleware(thunk)));
+    wrapper = mount(
+      <Provider store={store}>
+        <FoodList />
+      </Provider>
+    );
+  });
+
+  afterEach(() => {
+    //This is necessary to wait until your request has finished.
+    //Otherwise your test will try to finish without waiting for the request to finish.
+    wrapper.unmount();
+  });
+
+  it("Check if loading is true when retrieving info from the api", () => {
```

Comment:
`it('is loading foods')`

I'd try to follow this conventions taken from the BetterSpecs Ruby guides: http://www.betterspecs.org/